com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/BundleHelper.java

```diff
@@ -905,7 +905,7 @@ public static void extractPipelinePlugins(Project project, String pluginsDir) th
         ExtenderUtil.storeResources(new File(pluginsDir), sources);
     }
 
-    public static boolean isArchiveExcluded(Project project) {
+    public static boolean isArchiveIncluded(Project project) {
```

Comment:
This function means the opposite it returns. I renamed it to make sure it's clear